src/transpiling/mod.rs

```diff
@@ -138,21 +138,11 @@ impl EmitOptions {
 #[derive(Debug)]
 pub struct SourceMapConfig {
   pub inline_sources: bool,
-  /// When a base is provided, when mapping source names in the source map, the
-  /// name will be relative to the base.
-  pub maybe_base: Option<ModuleSpecifier>,
 }
 
 impl crate::swc::common::source_map::SourceMapGenConfig for SourceMapConfig {
   fn file_name_to_source(&self, f: &FileName) -> String {
-    match f {
-      FileName::Url(specifier) => self
-        .maybe_base
-        .as_ref()
-        .and_then(|base| base.make_relative(specifier))
```

Comment:
This does `"file:///file.js".make_relative("file:///file.js") === ""`

Reply:
I don't understand why someone was providing a base that was a filename. This code looks perfectly fine to me.